nima/webclient/webclient/src/main/java/io/helidon/nima/webclient/http1/Http1Client.java

```diff
@@ -35,11 +40,153 @@ static Http1ClientBuilder builder() {
+         *
+         * @param maxHeaderSize maximum header size
+         * @return updated builder
+         */
+        @ConfiguredOption("16384")
+        public Http1ClientBuilder maxHeaderSize(int maxHeaderSize) {
+            this.maxHeaderSize = maxHeaderSize;
+            return this;
+        }
+
+        /**
+         * Configure the maximum allowed length of the status line from the response.
+         *
+         * @param maxStatusLineLength maximum status line length
+         * @return updated builder
+         */
+        @ConfiguredOption("256")
+        public Http1ClientBuilder maxStatusLineLength(int maxStatusLineLength) {
+            this.maxStatusLineLength = maxStatusLineLength;
+            return this;
+        }
+
+        /**
+         * Sets whether Expect-100-Continue header is sent to verify server availability for a chunked transfer.
+         * <p>
+         *     Defaults to {@code true}.
+         * </p>
+         *
+         * @param sendExpect100Continue whether Expect:100-Continue header should be sent on chunked transfers
+         * @return updated builder
+         */
+        @ConfiguredOption("true")
+        public Http1ClientBuilder sendExpect100Continue(boolean sendExpect100Continue) {
+            this.sendExpect100Continue = sendExpect100Continue;
+            return this;
+        }
+
+        /**
+         * Sets whether the header format is validated or not.
+         * <p>
+         *     Defaults to {@code true}.
+         * </p>
+         *
+         * @param validateHeaders whether header validation should be enabled
+         * @return updated builder
+         */
+        @ConfiguredOption("true")
+        public Http1ClientBuilder validateHeaders(boolean validateHeaders) {
+            this.validateHeaders = validateHeaders;
+            return this;
+        }
+
+        /**
+         * Configure the default {@link MediaContext}.
+         *
+         * @param mediaContext media context for this client
+         * @return updated builder
+         */
+        @ConfiguredOption("io.helidon.nima.http.media.MediaContext")
+        public Http1ClientBuilder mediaContext(MediaContext mediaContext) {
+            Objects.requireNonNull(mediaContext);
+            this.mediaContext = mediaContext;
+            return this;
+        }
+
+        /**
+         * Configure the maximum allowed size of the connection queue.
+         *
+         * @param connectionQueueSize maximum connection queue size
+         * @return updated builder
+         */
+        @ConfiguredOption("256")
+        public Http1ClientBuilder connectionQueueSize(int connectionQueueSize) {
+            this.connectionQueueSize = connectionQueueSize;
+            return this;
+        }
+
+        /**
+         * Maximum allowed header size of the response.
+         *
+         * @return maximum header size
+         */
+        public int maxHeaderSize() {
```

Comment:
Should these methods be public? Isn't it enough to have those as package private?

Reply:
Good point... update and changed all the getter methods in this class as package local.